### PR TITLE
DEVX-1032 refactor ksql script handling into mapped files

### PR DESCRIPTION
**Pull request by rspurgeon**
This was my solution for running the ksql loading portion of the demo.  Added a few addl scripts which moved most of the processing into the docker container, which I hope makes things a little cleaner.

I targted DEVX-1032 first to isolate this change in a new PR, and then it will all be merged into 5.4.x after

### CODE REVIEW

File: scripts/ksql/run_ksql_commands.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+input=/tmp/ksqlcommands
+while IFS= read -r line
+do
+  /tmp/run_ksql_cmd.sh "$line"
```
Comment (ybyzek) [inline scripts/ksql/run_ksql_commands.sh]: @rspurgeon why require `run_ksql_cmd.sh`?  Why not just put here the content of `run_ksql_cmd.sh`?

Comment (rspurgeon) [reply scripts/ksql/run_ksql_commands.sh]: > @rspurgeon why require `run_ksql_cmd.sh`? Why not just put here the content of `run_ksql_cmd.sh`?

IMO the small files are clearer to read, and the single function "run_ksql_cmd" is potentially useful independently.  I could do a function in the script also, but this works.  If you want it smashed up into a single line i'll change it.  LMK

Comment (ybyzek) [reply scripts/ksql/run_ksql_commands.sh]: IMO it's better to mash it to avoid following a trail of files.   Thanks!

Comment (rspurgeon) [reply scripts/ksql/run_ksql_commands.sh]: Functional programmers might not agree ;-). I'll push a change

**Pull request merged**